### DISCUSSION TITLE
ScriptVimeoPlayer is loading video okay, but is not playing on click

### DISCUSSION THREAD

**Discussion by hanesmc** [nuxt/scripts#157] [Q&A]
I'm trying to use the ScriptVimeoPlayer. It's displaying the Vimeo video, but is not actually playing the video when clicked. This is happening not only in my application but on the Demo app on the Nuxt/Scripts site as well. 

This is my first time using these scripts, so it could easily be something I'm doing wrong. I was successful using the ScriptYouTubePlayer so I believe my install is good.

Here is the component syntax I am using.
<ScriptVimeoPlayer :id="idNumber" class="group" />

Any suggestions?

**Comment by harlan-zw** [comment]
I think the Vimeo player was bugged previously. The [demo](https://scripts.nuxt.com/scripts/content/vimeo-player) appears to be working now for me but maybe you can confirm on your end when you have a chance :pray:

**Comment by harlan-zw** [comment]
Closing as outdated — the [demo](https://scripts.nuxt.com/scripts/content/vimeo-player) has been working for a long while. Please open a fresh issue with a repro if you're still hitting this 🙏
